BEGINNERS.md

```diff
@@ -0,0 +1,309 @@
+# Minimal Snowdrift set up instructions
+
+This is a small tutorial on contributing to Snowdrift. You don't have to
+be tech savvy, but it helps. This is a minimal-ish description of what
+you need to do to get Snowdrift running locally and to start making and
+submitting updates.
+
+Although we recommend deeper study to understand how all of this works,
+you *can* simply follow the steps below, and it will work well enough
+to get started.
+
+Advanced users should adapt any of these instructions as seen fit.
+
+These instructions assume you use Debian testing or some variant of Ubuntu.
+Our general [GUIDE](GUIDE.md) has some notes for other systems.
+
+If you need *any* help at any point, come say "hi" at our freenode.net IRC
+channel [#snowdrift](https://snowdrift.coop/p/snowdrift/w/irc).
+We are always happy to assist and answer *any* questions!
+
+## Installing
+
+Open up a terminal, and run these commands, exactly as you see them. You
+could even copy and paste all of them into your terminal at once, and it
+would work.
+
+    sudo -i
+    aptitude update
+    aptitude install curl git postgresql postgresql-client
+    cd /usr/lib/x86_64-linux-gnu
+    ln -s libgmp.so.10 libgmp.so
+    exit
+    mkdir builds
+    cd builds
+    curl -ssL \
+      https://www.haskell.org/ghc/dist/7.8.4/ghc-7.8.4-x86_64-unknown-linux-deb7.tar.xz |
+      tar xJv
+    cd ghc-7.8.4
+    ./configure && make && sudo make install
```

Comment:
@singpolyma It's not building from source, it's a binary distribution. The latest GHC version in Debian unstable is 7.6.something, which is criminally old.